imports/api/demohouse.js

```diff
@@ -1036,33 +1036,45 @@ export function insertDemoHouse(lang) {
 }
 
 function deleteDemoUserWithRelevancies(userId, parcelId, communityId) {
+  const counter = Number(Meteor.users.findOne({ _id: userId }).emails[0].address.split('.')[0]);
   Topics.remove({ userId });
   // votes?  
-  // topics category: room, participantIds[]: userId
   Topics.remove({ 'participantIds.$': userId });
   Comments.remove({ userId });
   Delegations.remove({ sourceUserId: userId });
   Delegations.remove({ targetUserId: userId });
   Memberships.remove({ parcelId }); // removing added benefactors as well
   Parcels.remove({ _id: parcelId });
   const currentTotalunits = Communities.findOne({ _id: communityId }).totalunits;
-  Communities.update({ _id: communityId }, { $set: { totalunits: (currentTotalunits - 300) } });
+  if (currentTotalunits > 10000) {
+    Communities.update({ _id: communityId }, { $set: { totalunits: (currentTotalunits - 100) } });
+  }
+  ParcelBillings.remove({ 'accounts.Könyvelés helye': (14 + counter).toString() });
```

Comment:
Ize.... Ilyen methodust meg nem csinaltam. Csinaljak?
Jelenleg nincs mod visszacsinalni egy csoportos megbizast egyben.
Es nem is taroljuk oket, hanem csak applikaljuk, es csinalunk belole sok egyenkenti payment-et.

Reply:
Igazából itt a demoházban nincs jelentősége, mert úgyis töröljük az összes paymentet is, csak mindent törölni akarunk, ami bárhogy is a gombos demouserhez kapcsolódik.